src/cmd/manage_nsfs.js

```diff
@@ -252,7 +252,7 @@ async function delete_bucket(data, force) {
         }
         await native_fs_utils.folder_delete(bucket_temp_dir_path, fs_context_fs_backend, true);
         await config_fs.delete_bucket_config_file(data.name);
-        return { code: ManageCLIResponse.BucketDeleted, detail: '', event_arg: { bucket: data.name } };
+        return { code: ManageCLIResponse.BucketDeleted, detail: `Bucket ${data.name} has been successfully deleted.`, event_arg: { bucket: data.name } };
```

Comment:
I don't think it makes sense to add a message on every bucket operation response, we should add a this to the manage_nsfs_cli_responses file and set message for every op, we do the same on manage_nsfs_cli_errors 
and please add to all operations and not only to bucket/account delete

Reply:
Please correct me if I am wrong, if we add message on every bucket operation(then it should be something general like `Account deleted successfully.`). Here, still we need to customize it to put account/bucket etc. name in the message description.
As per the issue, we want more info on which account/bucket etc. was deleted and not a general description.

Reply:
the bucket name/account can be added via the argument, this issue is just an example of a bigger problem and I don't see a reason for not solving it for all the cases

Reply:
Sure, I will try apply this to all operations. Thanks

Reply:
Updated the code, please check now. Thanks